static/src/javascripts-legacy/projects/common/modules/experiments/tests/membership-engagement-banner-tests.js

```diff
@@ -78,5 +77,114 @@ define([
         return this.addMessageVariant(variantId, {contributions: variantParams});
     };
 
-    return []
+    function completer(complete) {
+        mediator.on('membership-message:display', function () {
+            // When the button link is clicked, call the function that indicates the A/B test is 'complete'
+            // ...note that for Membership & Contributions this completion is only the start of a longer
+            // journey that will hopefully end pages later with the user giving us money.
+            bean.on(qwery('#membership__engagement-message-link')[0], 'click', complete);
```

Comment:
The point of connection between JS and HTML should normally be a class starting with `js-` rather than an id. So the selector might be `.js-engagement-banner-link`

This is to distinguish classes which have behavioural significance from those which just have styling significance - ids are generally not used for anything (except possibly integration tests)